webdriver/tests/bidi/integration/cookies_with_network_events.py

```diff
@@ -194,8 +203,10 @@ async def test_image(
     )
     await wait_for_future_safe(on_before_request_sent)
 
+    image_path = os.sep + os.path.join(*(urlparse(image_url).path.split("/")[1:-1]))
```

Comment:
What about just replacing all slashes with `os.sep`? That's faster and easier to read as well.

```suggestion
    image_path = os.path.dirname(urlparse(image_url).path.replace("/", os.sep))
```